### DISCUSSION TITLE
multiline support for kubernetes_logs source

### DISCUSSION THREAD

**Discussion by gwang-JNPR** [vectordotdev/vector#20943] [Q&A]
Hi All,

I have java program running on k8s, I wish to combine multiline exceptions.
I know `file` source support multiline and I am currently using it: https://vector.dev/docs/reference/configuration/sources/file/#multiline

How to do the same via `kubernetes_logs` source?
transforms via Lua?

**Comment by gwang-JNPR** [comment]
found the answer: https://github.com/vectordotdev/vector/discussions/13080

**Comment by huanghuangzym** [comment]
can reduce support reg match?
